src/python_testing/global_attribute_ids.py

```diff
@@ -38,3 +39,95 @@ def to_name(self) -> str:
             return "FeatureMap"
         if self == GlobalAttributeIds.CLUSTER_REVISION_ID:
             return "ClusterRevision"
+
+
+@dataclass
+class IdRange:
+    min: int
+    max: int
+
+    def _contains(self, id: int):
+        return id in range(self.min, self.max+1)
+
+
+@dataclass
+class PrefixIdRange(IdRange):
```

Comment:
why is this a dataclass but SuffixIdRange is not?

Reply:
copy/paste error.